tests/end_to_end/end_to_end.cpp

```diff
@@ -3215,3 +3215,120 @@ TEST_CASE("multiple_map_insert", "[close_cleanup]")
 
     bpf_object__close(unique_object.release());
 }
+
+void
+test_no_limit_map_entries(ebpf_map_type_t type, bool max_entries_limited)
+{
+    uint32_t max_entries = 2;
+    fd_t inner_map_fd = ebpf_fd_invalid;
+    fd_t map_fd = ebpf_fd_invalid;
+    bool nested_map = (type == BPF_MAP_TYPE_HASH_OF_MAPS);
```

Comment:
Why does this type check get hard coded get here but others have macros in 3231-3233?

Reply:
Fixed.